iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/VoltageLevelAdapter.java

```diff
@@ -511,7 +511,22 @@ public StaticVarCompensatorAdder newStaticVarCompensator() {
     @Override
     public Iterable<Switch> getSwitches() {
         return Iterables.transform(getDelegate().getSwitches(),
-                                   getIndex()::getSwitch);
+                getIndex()::getSwitch);
+    }
+
+    @Override
+    public LccConverterStationAdder newLccConverterStation() {
+        return new LccConverterStationAdderAdapter(getDelegate().newLccConverterStation(), getIndex());
+    }
+
+    @Override
+    public Iterable<LccConverterStation> getLccConverterStations() {
+        return getLccConverterStationStream().collect(Collectors.toSet());
```

Comment:
Is the set needed? Is the order important?

Reply:
My mistake, i willl change it to List